client/src/components/Upload.vue

```diff
@@ -282,7 +272,7 @@ export default {
           const subfile = files.splice(0, 1);
           const subname = subfile[0].file.name.replace(/\..*/, '');
           // Only video subfolders should be used typically
-          const subtype = videoFilesRegEx.test(subfile[0].file.name) ? 'video' : 'unknown';
+          const subtype = this.vidRegEx.test(subfile[0].file.name) ? 'video' : 'unknown';
```

Comment:
Is this supposed to be `this.getVidRegEx`?

Reply:
It would appear so.

Reply:
@mattrix27 I think this is the last thing to check before merging.

Reply:
Oh yeah, it is supposed to